phpstan-baseline.neon

```diff
@@ -560,6 +560,11 @@ parameters:
 			count: 1
 			path: src/lib/Form/Type/RichTextFieldType.php
 
+		-
+			message: "#^Variable \\$element in PHPDoc tag @var does not match any variable in the foreach loop\\: \\$childNode$#"
+			count: 1
+			path: src/lib/RichText/Converter/ListLineBreaks.php
```

Comment:
FYI : Added for the same reasons as d3ac5fad35fd219f5b958557e19d7069759df18c

Reply:
> FYI : Added for the same reasons as [d3ac5fa](https://github.com/ezsystems/ezplatform-richtext/commit/d3ac5fad35fd219f5b958557e19d7069759df18c)

This is not related to d3ac5fad35fd219f5b958557e19d7069759df18c. 

If you change the relevant code to state:
```php
        /** @var \DOMElement $element */
        foreach ($elements as $element) {
```
the issue will disappear. Otherwise PHPStan seems to assume you're trying to type-hint the nested `foreach` right after that statement.

Reply:
ok. Thanx